app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragment.kt

```diff
@@ -18,8 +47,134 @@ class MoreFragment : NavigableFragment<MoreFragmentNavigator>() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
-    ) = MoreFragmentBinding.inflate(inflater, container, false).apply {
-        lifecycleOwner = viewLifecycleOwner
-        viewModel = moreViewModel
-    }.root
+    ) = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        setContent {
+            BenchProjectTheme {
```

Comment:
Minor: Would be nice to extract all the composables to another file, for example, `MoreScreen.kt`, then you will just call setContent { MoreScreen() } here. Thanks to that all the logic connected with the fragment will be separated from composables view (like for XML :D).

Reply:
Done :D